src/components/input/Input.tsx

```diff
@@ -46,9 +43,7 @@ export const Input = ({
   error,
   helpText,
   inputMode,
-  inputPrefix,
```

Comment:
This was a necessary work-around for the `<NumberInput />` in Dashboard. It looks like we are only using the `<NumberInput />` in a few places that are older. One option is to kill the `<NumberInput />` in Dashboard, but that may be a rabit hole.

```
/**
 * @desc An input that takes a number & returns a number
 * @tutorial https://github.com/s-yadav/react-number-format
 * @param `prefix` uses `react-number-format` prefix
 * @param `suffix` uses `react-number-format` suffix
 * @param `inputPrefix` uses `@zonos/amino` prefix
 * @param `inputSuffix` uses `@zonos/amino` suffix
 */
export const NumberInput = <T extends boolean = false>({
```

Reply:
I understand the repercussions in dashboard and will have a separate PR to fix the resulting issue there. I don't think that logic belongs in amino.

Reply:
May I ask what your plan is? `suffix` and `prefix` props are absorbed by the `react-number-format` package, which would deem a different prop necessary.

Reply:
My idea is to just move this logic into dashboard with a wrapper component.